Bindings/Java/Matlab/Hopper_Device/RunHopperWithDevice_answers.m

```diff
@@ -119,9 +119,9 @@
 reporter.setName('hopper_device_results');
 reporter.set_report_time_interval(0.2); % seconds.
 reporter.addToReport(...
-    hopper.getComponent('slider/yCoord').getOutput('value'), 'height');
+    hopper.getComponent('jointset/slider/yCoord').getOutput('value'), 'height');
 reporter.addToReport(...
-    hopper.getComponent('vastus').getOutput('activation'), 'vastus_activation');
+    hopper.getComponent('forceset/vastus').getOutput('activation'), 'vastus_activation');
```

Comment:
Same question for all sets, is that just a side effect of using an old model file that introduces the sets or is the layout now required? Thanks

Reply:
Not required. Just consequence of previous format and if using `Model::add##ComponentType##()`

Reply:
Thanks @aseth1 Do you think it would be confusing to users to have different paths depending on what call is used to add (e.g. addMarker, addComponent, addModelComponent)?

Reply:
It is a bit late to be asking this question. We agreed to maintain the Sets primarily to support navigation in the GUI. The only reason to have these Sets is for the GUI and the only reason `Model::add##ComponentType##()` are necesssary is to populate these sets so the added components appear in the GUI.

If you created Sets on the fly using `getComponentList<C>` then we wouldn't need them on the API side and the change to make Sets ModelComponents so they appear on the pathname would not be necessary. That being said, the transition from 3.3 to 4.0 would be more difficult with non unique names.

Reply:
@aseth1 For the record, the GUI does NOT use the Sets in the model directly, instead it uses iterators to populate a cached "Set" and that's what's used to back the navigator.

Reply:
Awesome. So we can remove all the Sets from `Model` and not have to support the multiple `add<ModelComponent>()` methods.  Sweet. That would simplify the `Model` interface a lot. If this is the case that makes this PR rather unnecessary, right?

Reply:
If we removed `Model`'s `Set`s then we would have to deal with renaming in the case of duplicates, yes, but this PR wouldn't make make sense since we would not have `BodySet`, `ForceSet`, .... As it stands, we still have a mix of old (`addBody`, `addJoint`, ...) and new `addComponent()`, which as you point out is confusing.